app/search/models.py

```diff
@@ -56,7 +53,14 @@ def get_data(self):
         for result in results:
             data = result.__dict__
             data['descriptor'] = data['descriptor']
-            data.pop('_sa_instance_state', None)
             data['status'] = data['status'].value
+            p = result.publisher
+            data['publisher_name'] = p.name
+            if '_sa_instance_state' in data:
+                data.pop('_sa_instance_state', None)
```

Comment:
@Fak3 i know. @rufuspollock told me return back all fields as right now we do not know how many fields we want. Will improve later.